__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../Endpoints/app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("app", () => {
+  describe("GET /api/topics", () => {
+    test("200: Should respond with an array of objects with slug and description keys", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          response.body.topics.forEach((topic) => {
```

Comment:
when running array methods (that eval to true) in our test suite, we should _always_ check that the length of the array is greater than 0, because otherwise, the array method automatically returns true (and therefore our tests auto-pass).